src/osdp_cp.c

```diff
@@ -654,6 +654,12 @@ static int cp_decode_response(struct osdp_pd *pd, uint8_t *buf, int len)
 		sc_activate(pd);
 		ret = OSDP_CP_ERR_NONE;
 		break;
+        case REPLY_ISTATR:
+		/* Ignore unsupported reply without creating error */
+		LOG_WRN("Ignoring unsupport REPLY(%02x)", pd->reply_id);
+		ret = OSDP_CP_ERR_NONE;
+                break;
+
```

Comment:
Perhaps we should have a flag IGNORE_UNSUPPORTED on a PD level to do this. There can be other PDs that report other things we don't care about. 

This is okay for now. I'll make the flags change change.